forgerock-auth/src/androidTest/java/org/forgerock/android/auth/MockServer.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2021 ForgeRock. All rights reserved.
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+package org.forgerock.android.auth;
+
+import android.content.Context;
+
+import androidx.test.core.app.ApplicationProvider;
+
+import com.squareup.okhttp.mockwebserver.MockResponse;
+import com.squareup.okhttp.mockwebserver.MockWebServer;
+
+import org.apache.commons.io.IOUtils;
+import org.junit.After;
+import org.junit.Before;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+
+public abstract class MockServer {
+
+    protected MockWebServer server;
+
+    public Context context = ApplicationProvider.getApplicationContext();
+
+    @Before
+    public void startServer() throws Exception {
+
```

Comment:
Suggest remove extra empty lines to be consistent with the coding style in the rest of the class.